### DISCUSSION TITLE
In network files: thousands of nonsensical billing codes per provider?

### DISCUSSION THREAD

**Discussion by quadrupole** [CMSgov/price-transparency-guide#761] [Q&A]
Have people found that there are large quantities of billing codes that are not relevant for a particular provider? For example, looking at one provider in particular, I see over 100,000 billing codes with in network rates which are clearly not relevant (different treatment area and unlikely that they would have negotiated so many rates for a single payor. I am trying to understand if:
1. This is expected - if so what would explain this?
2. Have I made some error in my data parsing - so far I cant find it

Any help/comment would be much appreciated!

**Comment by rgiljohann** [comment]
This is happening in the data, you are not seeing an error. This is a byproduct in how rates are actually negotiated vs. how the Price Transparency Rule is implemented. The rule requires every possible billing code to be listed. This is why you get the huge number.

In reality, rates are negotiated differently. Lets take CPT codes for example. Most of the time, groups of CPT codes are negotiated together. For example, all Evaluation Codes are grouped together (99213, 99214, etc...). They could be all set to a % of CMS rates (which is a very common practice in rate negotiations. 

So in the end, the negotiated contract will only have 20-30 negotiated rates (those different groupings of codes with a % of CMS), that encompass all possible codes (> 10,000 CPT codes). The rule requires them to list out every possible code. 

Not an easy solution to fix, but the rule does not necessarily correlate to how rates are negotiated.

**Reply by quadrupole** [reply]
Thanks for your reply! Just want to play back my understanding:

> The rule requires every possible billing code to be listed.

But I dont understand why this necessitates that every NPI should have every billing code attached to it for that sake. Wouldnt it be clearer and not to mention more storage efficient, if the MRFs were organised such that every billing code is listed, but only those NPIs that had negotiated a particular rate for that billing code were attached to it? I dont understand why the present schema was chosen since it clearly results a ton of unnecessary information.

**Comment by bryanbriegel** [comment]
Right -the TiC rule requires all rates that are part of the contract with providers to be exposed in the MRFs, and that can add a lot of codes listed for a provider that are outside of their scope of practice or implausible.   Where contracts are more specific and limited to a provider's scope of practice we'd not see this in the MRFs.

**Reply by quadrupole** [reply]
Thanks for your reply! So essentially we cannot use the MRFs to determine how many rates a particular provider is billing, but for those rates that they do bill, the correct negotiated rate will be present in the MRF? To me its a strange way to organize the data - it clearly leads to a lot of redundancy, wouldnt it have been better to list out objects of billing codes at different negotiated rates, and have them map to groups of providers, with many providers not being part of the group for which they would not have negotiated a rate? This would definitely reduce file sizes.
